src/content/docs/accounts/accounts-billing/new-relic-one-user-management/user-management-ui-and-tasks.mdx

```diff
@@ -22,7 +22,7 @@ The easiest way to see what your user management permissions are is to go to the
 Some user management requirements and restrictions:
 
 * This doc is about managing users on the [New Relic One user model](/docs/accounts/original-accounts-billing/original-product-based-pricing/overview-user-models). 
-* Most user management capabilities require the [**Authentication domain manager** role](/docs/accounts/accounts-billing/new-relic-one-pricing-users/users-roles#standard-roles) and some require the [**Organization manager** role](/docs/accounts/accounts-billing/new-relic-one-pricing-users/users-roles#standard-roles). For details, see [Standard roles](/docs/accounts/accounts-billing/new-relic-one-user-management/user-management-concepts#standard-roles).
+* Most user management capabilities require the [**Authentication domain manager** role](/docs/accounts/accounts-billing/new-relic-one-user-management/user-management-concepts#standard-roles) and some require the [**Organization manager** role](/docs/accounts/accounts-billing/new-relic-one-user-management/user-management-concepts#standard-roles). For details, see [Standard roles](/docs/accounts/accounts-billing/new-relic-one-user-management/user-management-concepts#standard-roles).
```

Comment:
@zuluecho9, the Gatsby preview suddenly stopped working, so I switched to a local build. I'm afraid that the "Organization manager" link never loaded the page: http://localhost:8000/docs/accounts/accounts-billing/new-relic-one-user-management/user-management-concepts#standard-roles.